BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -372,6 +377,8 @@
   }
 !endif
 
+  PayloadPkg/CsmeUpdateDriver/CsmeUpdateDriver.inf

```

Comment:
I Think this should be in FirmwareUpdate.inf as one of LibraryClasses. Only FirmwareUpdate payload use this, right? Better to have name CsmeUpdateLib instead of Driver.

Reply:
This is not a library, this is a driver that consumes a library. Since this is part of payload package, it is agreed to keep it here. Also new code has some changes coming to this